### DISCUSSION TITLE
[Run issue]error while loading shared libraries: libQt53DExtras.so.5: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by LouisLechNissen** [OpenTOPAS/OpenTOPAS#42] [Run-Problem] [answered]
### Contact Details

lonis@dtu.dk

### What was the issue?

Hi everyone!

I'm trying to install OpenTOPAS on a linux machine (Ubuntu 22.04). Installation seems to go okay until I try to run the QtShapeTest.txt-example. I'm following this guide: https://opentopas.readthedocs.io/en/latest/getting-started/Debian.html and step 9.1.b. where the shellScript is created. I've tried using step 9.1.a and it doesn't seem to work either.

This is the error message:
![image](https://github.com/OpenTOPAS/OpenTOPAS/assets/111356938/26ea0dd6-4185-4b28-9210-cbd67c23220f)

However, there seems to be a "libQt53DExtras.so.5" in an anaconda3-folder: 
![image](https://github.com/OpenTOPAS/OpenTOPAS/assets/111356938/f8e640c2-8013-489f-997f-f8496105a463)

To be honest I don't know what to do now. What would be the best approach here?

Thank you in advance.

Best regards,

Louis

### TOPAS category

Outcome model

### Version

4.0

### What Operative System (OS) are you running on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [ ] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by ghost** [comment, marked as answer]
Hello!

I am not an expert here so please correct me if I am wrong, but I experienced the exact same problem previously and I have solved it. 

From my experience, I learnt that it might have something to do during the Geant4 installation. 
Instead of running the command cmake (blablabla) try use ccmake (blablabla) instead. It will open a GUI like this
![image](https://github.com/OpenTOPAS/OpenTOPAS/assets/152828218/ed17537e-fd2e-4043-81b3-444cc48677c9)
(You will need to configure once or twice before those lists show up)

After that you need to find your computer Qt lib, not the anaconda lib. This is my correct directory
![image](https://github.com/OpenTOPAS/OpenTOPAS/assets/152828218/aa6e4d94-0c81-4454-a158-893ac115b57d)

You need to copy the location and directly change the anaconda lib into your computer lib. I didn't have the corrected picture, but basically you only have to replace the (in my case) `/home/cecilia/anaconda3/lib/` into `/usr/lib/x86_64-linux-gnu/` and continue with your installation

I am not sure what is the correct directory will be for you, but I hope this helps!

**Reply by LouisLechNissen** [reply]
That did the trick!  Thank you Cecilia!

Have a wonderful day, and happy TOPAS'ing :)

Best regards,

Louis

**Reply by ghost** [reply]
Hi Louis,

I'm glad to hear it worked! Thank you for letting me know.

Hope you will also have a wonderful day ^^

Best,
Cecilia

**Comment by jschuemann** [comment]
Hi Cecilia,

thank you for helping out answering questions on the forum, we really appreciate our users helping other users!
Best
      Jan

**Reply by ghost** [reply]
Hi Jan,

Thank you! I'm happy to help and contribute to the community.

Best,
Cecilia
